bin/ream/src/cli/beacon_node.rs

```diff
@@ -98,6 +101,7 @@ pub struct BeaconNodeConfig {
 impl From<BeaconNodeConfig> for ManagerConfig {
     fn from(config: BeaconNodeConfig) -> Self {
         Self {
+            current_epoch: config.current_epoch,
```

Comment:
please remove this, the current_epoch shouldn't be an argument, you can calculate it based off the time from genesis. every 32 slots is 1 epoch. a slot happens every 12 seconds